File: android-pdf-viewer/src/main/java/com/infomaniak/lib/pdfview/PDFSpacing.java
package com.infomaniak.lib.pdfview;

public class PDFSpacing {
    /** Fixed spacing between pages in pixels */
    int pageSeparatorSpacing;
    /** Fixed spacing between pages in pixels */
    int startSpacing;
    /** Fixed spacing between pages in pixels */
    int endSpacing;
    /** Calculate spacing automatically so each page fits on it's own in the center of the view */
    boolean autoSpacing;

    public PDFSpacing(int pageSeparatorSpacing, int startSpacing, int endSpacing, boolean autoSpacing) {
        this.pageSeparatorSpacing = pageSeparatorSpacing;
        this.startSpacing = startSpacing;
        this.endSpacing = endSpacing;
        this.autoSpacing = autoSpacing;
    }
}

File: android-pdf-viewer/src/main/java/com/infomaniak/lib/pdfview/PDFSpacing.kt
package com.infomaniak.lib.pdfview

data class PDFSpacing(
    /** Fixed spacing between pages in pixels  */
    val pageSeparatorSpacing: Int = 0,
    /** Fixed spacing between pages in pixels  */
    val startSpacing: Int = 0,
    /** Fixed spacing between pages in pixels  */
    val endSpacing: Int = 0,
    /** Calculate spacing automatically so each page fits on it's own in the center of the view  */
    val autoSpacing: Boolean = false
)
